### PR TITLE
Add shumvgolove to CODEOWNERS

### PR DESCRIPTION
**Pull request by tcely**
I don't know if you want to do this, so feel free to close this PR if you don't.

It would have been better to have @shumvgolove added automatically as a reviewer when editing the docker entrypoint file.

### CODE REVIEW

**Comment by shumvgolove** [comment]
Thanks, this is long overdue.

File: .github/CODEOWNERS [hunk 1]
```diff
@@ -1,3 +1,4 @@
 * @epoberezkin @efim-poberezkin
-/*Dockerfile @shumvgolove @epoberezkin @efim-poberezkin
+/Dockerfile @shumvgolove @epoberezkin @efim-poberezkin
```
Comment (tcely) [inline .github/CODEOWNERS]: There isn't a single `Dockerfile` in the root of this repository currently.

File: .github/CODEOWNERS [hunk 2]
```diff
@@ -1,3 +1,4 @@
...
 /scripts/docker/ @shumvgolove @epoberezkin @efim-poberezkin
+/scripts/main/ @shumvgolove @epoberezkin @efim-poberezkin
```
Comment (tcely) [inline .github/CODEOWNERS]: The `scripts` directly doesn't have a `main` subdirectory in this repository yet.

File: .github/CODEOWNERS (hunk 1 above)
Comment (shumvgolove) [reply .github/CODEOWNERS]: Will be after this merged: https://github.com/simplex-chat/simplexmq/pull/750

File: .github/CODEOWNERS (hunk 2 above)
Comment (shumvgolove) [reply .github/CODEOWNERS]: Same as above.

**Pull request merged**